src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFsDriver.Linux.cs

```diff
@@ -402,17 +468,17 @@ private unsafe bool WasEventDetected(int pollFileDescriptor, int valueFileDescri
 
             while (!cancellationToken.IsCancellationRequested)
             {
-                // poll every 1 millisecond
+                // Wait until something happens
                 int waitResult = Interop.epoll_wait(pollFileDescriptor, out epoll_event events, 1, _pollingTimeoutInMilliseconds);
                 if (waitResult == -1)
                 {
-                    throw new IOException("Error while trying to initialize pin interrupts.");
+                    throw new IOException("Error while waiting for pin interrupts.");
                 }
 
                 if (waitResult > 0)
                 {
                     pinNumber = events.data.pinNumber;
-
+                    /*
```

Comment:
I don't remember 100% from the top of my head, but I believe this code was required in order to reset the interrupt from epoll and that is why we added it. Is that not the case? If not, we should probably just remove this code instead of having it commented out.

Reply:
I removed it because the code doesn't do anything (the values read are all discarded). Everything works fine without this, for all the tests I've done.
I'll remove it.

Reply:
It's true that all values read are discarded, but the important part of this code was that by running the lseek, we were effectively resetting the epoll set, so it was kind of marking the notification as read if you will, so I'm worried now that after removing this, we will continue to get hits in epoll_wait() event when we didn't really get a new event.

Reply:
I see your point. But wouldn't we see test failures if that was the case? Especially my FastInterruptTest would certainly detect many more events than expected. I'll make sure this would be detected.

Reply:
@joperezr I can't confirm your observation, or I wouldn't know how. All the tests pass whether this code is enabled or not. And I'm not getting extra events in any tests, it seems.

Reply:
I suppose doing this small operation doesn't really affect performance much, so would you mind keeping it for now and logging an issue to do much more deep investigation on whether or not we can remove it so that we do that appart from this PR?

Reply:
That's acceptable for me, since the preferred solution is now anyway to use libgpiod. I've created #1028 .

Reply:
@joperezr Done